frontend/src/scenes/organization/Invites/CreateOrgInviteModal.tsx

```diff
@@ -53,11 +55,11 @@ export function CreateOrgInviteModalWithButton(): JSX.Element {
                 visible={isVisible}
             >
                 <p>
-                    Create an invite link for a teammate with a specific email address.
+                    Create an invite for a teammate with a specific email address.
```

Comment:
I think this copy can be simplified a bit, I mainly want us to remove the language around "creating an invite [...] specific email address", seems to add strain to the mental model.

**If email is available**,
```
Your teammate will be able to join by clicking on the email they'll receive.
```

**If email is not available**,
```markdown
We'll create an invite link for your teammate. **Share it** with them so they'll be able to sign up.
```